packages/core/src/promotionEvaluations/redux/selectors.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @module promotionEvaluations/selectors
+ * @category Promotion Evaluations
+ * @subcategory Selectors
+ */
+
+import { getError, getIsLoading, getResult } from './reducer';
```

Comment:
A selector for getting the `id` in the reducer would be useful :)

Reply:
Totally forgot to add the selector after adding to the reducer. Added 🙂